doc/docstyle.rst

```diff
@@ -8,6 +8,8 @@ General Guidelines
   procs can be useful too (visible via ``nim doc --docInternal foo.nim``).
 * Within documentation, a period (`.`) should follow each sentence (or sentence fragment) in a comment block. The documentation may be limited to one sentence fragment, but if multiple sentences are within the documentation, each sentence after the first should be complete and in present tense.
 * Documentation is parsed as a custom ReStructuredText (RST) with partial markdown support.
+* Prefer single backtick to double backtick since it's simpler and nim doc supports it, even in rst files.
+* See also [nep1](https://nim-lang.github.io/Nim/nep1.html) which should probably be merged here.
```

Comment:
```suggestion
* See also `nep1 <https://nim-lang.github.io/Nim/nep1.html>`_ which should probably be merged here.
```

Wrong link syntax.

Reply:
I've now changed it to rst-style, but note that `nim rst2html` now supports the markdown link syntax